assets/aws/files/install-hardened.sh

```diff
@@ -60,3 +60,6 @@ rm -rf /var/cache/dnf /var/cache/yum
 # Enable Teleport services to start on boot
 systemctl enable teleport-generate-config.service
 systemctl enable teleport.service
+
+# Disable password age on ec2-user account to prevent users being locked out (V-230367)
+chage -M -1 ec2-user
```

Comment:
nit: could you move this up to where the teleport user is added so all the user setup stuff is done together, and use the long opt `--maxdays` instead of `-M`. I prefer long opts in scripts for uncommon commands or arguments - just a bit more readable.